spec/index.bs

```diff
@@ -2025,11 +2025,14 @@ enum GPUTextureFormat {
 };
 </script>
 
-The depth aspect of the `depth24plus` family of formats
-({{GPUTextureFormat/depth24plus}} and {{GPUTextureFormat/depth24plus-stencil8}})
-may be implemented as either a 24-bit unsigned normalized value ("depth24unorm")
+The depth aspect of the {{GPUTextureFormat/"depth24plus"}}) and {{GPUTextureFormat/"depth24plus-stencil8"}})
+formats may be implemented as either a 24-bit unsigned normalized value ("depth24unorm")
```

Comment:
```suggestion

Issue: add something on GPULimits that gives an estimate of the bytes per texel of "stencil8"
```

Reply:
I think this edit got inserted on the wrong line